Nav.php

```diff
@@ -164,6 +174,7 @@ public function renderItem($item)
         $items = ArrayHelper::getValue($item, 'items');
         $url = ArrayHelper::getValue($item, 'url', '#');
         $linkOptions = ArrayHelper::getValue($item, 'linkOptions', []);
+        $asDropdown = isset($item['inline']) ? !$item['inline'] : $this->asDropdown;
```

Comment:
I let different because they are different things. One is a global property of the widget, another is an option item.

You think it might be the same name?